spec_parser/main.py

```diff
@@ -3,12 +3,16 @@
 # SPDX-License-Identifier: Apache-2.0
 
 from spec_parser import Model
-from runparams import RunParams
+from spec_parser.runparams import RunParams
 
-if __name__ == "__main__":
+
+def main():
```

Comment:
why create a function? Is it something setuptools require?

Reply:
Yes; this is how you get a `spec-parser` command-line utility.  You have to identify a function for Python to call when run as a command-line tool.